src/parseSet.java

```diff
@@ -0,0 +1,71 @@
+package src;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.*;
+
+class parseSet {
+
+    private static final String DELIMITER = "[^a-zA-Z0-9_,]+";
+    private static final String FIRSTPATH = "Text/FIRST.txt";
+    private static final String PREDICTPATH = "Text/PREDICT.txt";
+    static Map<String, Map<String, Boolean>> FIRST = new HashMap<>();
+    static Map<String, Map<String, List<String>>> PREDICT = new HashMap<>();
+
+    /**
+     * Populate FIRST or PREDICT data structure for LL parsing given file with
```

Comment:
Why is this in its own file, as opposed to in the main body?

Reply:
There were alternatives for setting the parse sets so I wanted to separate the two in case we decided to change how we loaded the parse sets. Whats the benefit of having the two combined?

Reply:
Instead of dealing with file io, what do you propose? I think we could hardcode the HashMaps (using whats generated from parseSet) or embed the logic in each node.

Reply:
Actually, It may make more sense to use the files so that we don't have to delve deep for parts 2 and 3 (or for any errors in either parse list)

Reply:
It also keeps the sets centralized for modification